src/main/java/com/gmail/woosay333/onlinebookstore/controller/BookController.java

```diff
@@ -51,12 +59,14 @@ public BookDto updateBook(
 
     @ResponseStatus(HttpStatus.NO_CONTENT)
     @DeleteMapping("/{id}")
+    @Operation(summary = "Delete a book by id", description = "Delete a book by id")
     public void delete(@PathVariable Long id) {
         bookService.delete(id);
     }
 
     @GetMapping("/search")
-    public List<BookDto> search(BookSearchParameters searchParameters) {
-        return bookService.search(searchParameters);
+    @Operation(summary = "Search books by parameters", description = "Search books by parameters")
+    public List<BookDto> search(BookSearchParameters searchParameters, Pageable pageable) {
```

Comment:
we can add more details on the search parameters to the swagger, would be very useful!